pkg/cloudvolume/login.go

```diff
@@ -34,17 +34,22 @@ func newLoginCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&opts.host, "host", "", "Cloud Volumes portal hostname/ip")
-	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "cloudvolume username")
-	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "cloudvolume passowrd")
+	cmd.Flags().StringVar(&opts.host, "host", cvDefaultHost, "HPE Cloud Volumes portal hostname/ip")
+	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "HPE Cloud Volumes username")
+	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "HPE Cloud Volumes password")
 	cmd.Flags().BoolVarP(&opts.passwordStdin, "password-stdin", "", false, "read password from stdin")
-	_ = cmd.MarkFlagRequired("host")
 	_ = cmd.MarkFlagRequired("username")
 
 	return cmd
 }
 
 func validateArgs(opts *cvLoginOptions) error {
+
+
+	if opts.host == "" {
```

Comment:
I don't think this is needed... just like the logout, it should now default to the default host if nothing is entered.